<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scenariosWindow</class>
 <widget class="QDialog" name="scenariosWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenarios</string>
  </property>
  <widget class="QComboBox" name="selectScen_cBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectScen_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select scenario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="createScen_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create scenario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newScenName_lineEd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addScen_Butt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="delScen_Butt">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportScen_Butt">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>export</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="importScen_Butt">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>import</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="events_tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>181</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delEvent_Butt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>134</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>delete event</string>
   </property>
  </widget>
  <widget class="QPushButton" name="execScen_Butt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>execute</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
